From b4d4f850d9a7a2e4f8f2a5c3748495c6e65a0b56 Mon Sep 17 00:00:00 2001
From: Wang Panzhenzhuan <randy.wang@rock-chips.com>
Date: Thu, 14 Feb 2019 17:27:17 +0800
Subject: [PATCH] media: rockchip: isp1: fix high fps preview blurred bug

fix following camera err:
    rkisp1: CIF_ISP_PIC_SIZE_ERROR (0x00000002)

Change-Id: I6168d352d521cf75d7537ffe70a9af6f2ec07282
Signed-off-by: Wang Panzhenzhuan <randy.wang@rock-chips.com>
---
 drivers/media/platform/rockchip/isp1/isp_params.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/media/platform/rockchip/isp1/isp_params.c b/drivers/media/platform/rockchip/isp1/isp_params.c
index 1496ba4a8f23..49fef2967bfb 100644
--- a/drivers/media/platform/rockchip/isp1/isp_params.c
+++ b/drivers/media/platform/rockchip/isp1/isp_params.c
@@ -506,8 +506,6 @@ static void isp_goc_config_v10(struct rkisp1_isp_params_vdev *params_vdev,
 {
 	int i;
 
-	isp_param_clear_bits(params_vdev, CIF_ISP_CTRL,
-			     CIF_ISP_CTRL_ISP_GAMMA_OUT_ENA);
 	rkisp1_iowrite32(params_vdev, arg->mode, CIF_ISP_GAMMA_OUT_MODE_V10);
 
 	for (i = 0; i < params_vdev->config->gamma_out_max_samples; i++)
@@ -521,8 +519,6 @@ static void isp_goc_config_v12(struct rkisp1_isp_params_vdev *params_vdev,
 	int i;
 	u32 value;
 
-	isp_param_clear_bits(params_vdev, CIF_ISP_CTRL,
-			     CIF_ISP_CTRL_ISP_GAMMA_OUT_ENA);
 	rkisp1_iowrite32(params_vdev, arg->mode, CIF_ISP_GAMMA_OUT_MODE_V12);
 
 	for (i = 0; i < params_vdev->config->gamma_out_max_samples / 2; i++) {
-- 
2.35.3

